README.md

```diff
@@ -1,38 +1,84 @@
 # GOV.UK Helm Charts
 
+
+## Getting started
+
+See [Helm's documentation](https://helm.sh/docs) to get started with Helm.
+
+See the [GOV.UK Kubernetes Cluster User Docs] for an introduction to the
+cluster or ask [#govuk-replatforming] in Slack.
+
+[GOV.UK Kubernetes Cluster User Docs]: https://govuk-k8s-user-docs.publishing.service.gov.uk/
+[#govuk-replatforming]: https://gds.slack.com/channels/govuk-replatforming
+
+
 ## Local development
 
+Most of the charts in this repository are designed to be installed via [Argo
+CD] rather than by `helm`. We hope to improve portability over time.
+
+[Argo CD]: https://argo-cd.readthedocs.io/en/stable/
+
+### Prerequisites
+
+1. Install Helm and [yq](https://github.com/mikefarah/yq#yq)
+
+    ```sh
+    brew install helm yq
+    ```
+
 1. Clone the repository
-1. Configure git hooks: `git config core.hooksPath git-hooks`
 
-## Usage
+    ```sh
+    git clone git@github.com:alphagov/govuk-helm-charts.git
+    ```
+
+1. Configure git hooks
+
+    ```sh
+    cd govuk-helm-charts &&
+    git config core.hooksPath git-hooks
+    ```
+
 
-[Helm](https://helm.sh) must be installed to use the charts.  Please refer to
-Helm's [documentation](https://helm.sh/docs) to get started.
+### Installing an application chart without Argo CD
 
-Refer to the [GOV.UK Kubernetes Cluster User Docs] for an introduction to
-the cluster or ask for help from [#govuk-replatforming] in Slack.
+```sh
```

Comment:
Might be worth referencing which directory the user should be in

Reply:
Good shout, thanks!